src/document/DocumentCommandHandlers.js

```diff
@@ -528,6 +530,7 @@ define(function (require, exports, module) {
         return handleFileOpen(commandData).done(function (file) {
             var paneId = (commandData && commandData.paneId) || MainViewManager.ACTIVE_PANE;
             MainViewManager.addToWorkingSet(paneId, file, commandData.index, commandData.forceRedraw);
+            HealthLogger.fileOpened(file.fullPath, true);
```

Comment:
This way, we call `fileOpened` two times (one time for `handleFileOpen`, then for `handleFileAddToWorkingSetAndOpen`). Is it meant to be like this?